Formula/chamber.rb

```diff
@@ -0,0 +1,31 @@
+class Chamber < Formula
+  desc "CLI for managing secrets through AWS SSM Parameter Store"
+  homepage "https://github.com/segmentio/chamber"
+  url "https://github.com/segmentio/chamber.git",
+    :tag => "v1.9.0",
+    :revision => "acc4072df92f2763217ccfa5373e14f342e178ba"
+  sha256 "098568ddda04956b8dc270aa8e6b4eb83d8cfe92add8ea0172715eab7db050cb"
+  head "https://github.com/segmentio/chamber.git"
+
+  depends_on "go" => :build
+  depends_on "govendor" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = "amd64"
+    ENV["CGO_ENABLED"] = "0"
+
+    path = buildpath/"src/github.com/segmentio/chamber"
+    path.install Dir["{*,.git}"]
+
+    cd buildpath/"src/github.com/segmentio/chamber" do
+      system "govendor", "sync"
+      system "go", "build", "-o", bin/"chamber"
```

Comment:
Suspect you're going to need a `prefix.install_metafiles` here.

Reply:
I can add it, but I'm not entirely sure what it does. Is there documentation about what it's meant for?

Reply:
No, but given I get asked this question on an at least weekly basis I'm seriously contemplating adding some 🙈.

Essentially it ensures things like `README`, `LICENSE`, `CHANGELOG` are installed in the `prefix` where they exist. This is done automatically from the buildpath, but for most `go`-using formulae _everything_ is relocated from the `buildpath` into another directory which borks that logic. It's a known issue.

Reply:
textexpander.

Reply:
I still have some quick replies from my maintainer days saved on GitHub but they don't seem to work in this field 😭. That was what I used to rely on for frequent questions/thanks/etc.